src/beaconer.rs

```diff
@@ -58,7 +61,7 @@ pub struct Beaconer {
     /// The last beacon that was transitted
     last_beacon: Option<beacon::Beacon>,
     /// Use for channel plan and FR parameters
-    region_params: Option<RegionParams>,
+    region_params: RegionParams,
```

Comment:
what has changed that lets us avoid wrapping params in `Option` all over now?

Reply:
That first From<Region> for RegionParams allows us to construct a "default" RegionParams with no gain and empty params but a region set. This simplifies a lot at the expense of needing to know that empty params in RegionParams means you can't beacon or downlink